locale/en/blog/uncategorized/evolving-the-node-js-brand.md

```diff
@@ -32,6 +32,6 @@ We look forward to exploring<ins cite="mailto:EMILY%20TANAKA-DELGADO" datetime="
 
 We hope you'll have fun using it.
 
-To download the new logo, visit <a href="http://nodejs.org/logos/">nodejs.org/logos</a>.
+To download the new logo, visit <a href="https://nodejs.org/logos/">nodejs.org/logos</a>.
```

Comment:
This redirects now to https://nodejs.org/static/images/logos/ and returns a 403 (directory index is forbidden).
https://nodejs.org/en/about/resources/ would be a better match, but direkt links to the logo files should still go to static/.